src/views/virtualmachines/utils/virtualMachineRowFilter.ts

```diff
@@ -55,6 +57,28 @@ const useStatusFilter = (): RowFilter => ({
   items: statusFilterItems,
 });
 
+const useLiveMigratableFilter = (): RowFilter => {
+  const [isSingleNodeCluster] = useSingleNodeCluster();
+
+  return {
+    filterGroupName: t('Live migratable'),
+    type: 'live-migratable',
+    items: [
+      { id: 'yes', title: t('Yes') },
```

Comment:
can you please add 'yes' and 'no' to a constants file? for the id's only

Reply:
That's a good suggestion! I totally agree! Thanks! :)